src/intree.js

```diff
@@ -92,6 +92,15 @@ module.exports.setup = function(cfg) {
       'taskcluster.docker.workerType': cfg.intree.workerType,
     }));
 
+    // Temporary loop for debugging purposes
+    config.tasks.forEach(task => {
+      if (task.extra.github.events.some(event => event == 'release')) {
```

Comment:
You mean add it to try-catch? I didn't :) However, before messing up commits again, I deployed the thing, and it appears there may be something wrong with parsing YML. I'll look more closely later.